docs/cqs_aop_primer.md

```diff
@@ -0,0 +1,16 @@
+# Why use Functional.CQS.AOP?
+`Functional.CQS.AOP` makes it easy to apply cross-cutting concerns to [`Functional.CQS` handler implementations](https://github.com/RyanMarcotte/Functional.CQS/tree/master/src/Functional.CQS) by supplying pluggable abstractions and - when DI containers are involved - handling common bootstrapping.  The separation of cross-cutting concerns from business logic is [aspect-oriented programming](https://en.wikipedia.org/wiki/Aspect-oriented_programming) (AOP).
+
+An example CQS-based architecture is illustrated below:
+
+![example CQS architecture with AOP](images/CQS_architecture.png)
```

Comment:
This image's tag should omit the words `with AOP`.

Reply:
Fixed